### ISSUE TITLE
Initial normfactor settings and nominal model rates

### ISSUE THREAD

**Issue by alexander-held** [scikit-hep/pyhf#1106]
# Question

Should `pyhf.Model.nominal_rates` pick up initial values specified in the model config? At the moment it does not:

```python
import pyhf

spec = {
    "channels": [
        {
            "name": "SR",
            "samples": [
                {
                    "data": [10.0],
                    "modifiers": [
                        {"data": None, "name": "Signal_norm", "type": "normfactor"}
                    ],
                    "name": "Signal",
                },
                {"data": [50.0], "modifiers": [], "name": "Background"},
            ],
        }
    ],
    "measurements": [
        {
            "config": {
                "parameters": [
                    {"bounds": [[0, 5]], "inits": [2.0], "name": "Signal_norm"}
                ],
                "poi": "Signal_norm",
            },
            "name": "minimal_example",
        }
    ],
    "observations": [{"data": [65.0], "name": "Signal_region"}],
    "version": "1.0.0",
}

model = pyhf.Workspace(spec).model()
print(model.nominal_rates)
```
results in
```
[[[[50.]]

  [[10.]]]]
```

where the POI setting `"inits": [2.0]` is ignored. I intuitively interpret the init as the "nominal" setting. I would therefore expect it to be picked up by `nominal_rates`, since there is no other way to specify such a value as far as I am aware.

This issue is related to #1029. Here, I would find it convenient for inits of normfactors to be included. In #1029, the issue is that inits for other parameters make a dataset non-Asimov, and thus I would find it more natural to not make use of them.

Perhaps it is best to not change the existing behavior and for consistency to also ignore the inits for normfactors. In that case, what do you think about an extra setting to set nominal normfactor values? Those would add some flexibility in defining models that measure things that are not just ratios (with nominal value 1, which now is assumed).

# Relevant Issues and Pull Requests

#1029

**Comment by lukasheinrich** [comment]
hm, maybe we should rename this to `base_rates`. But we should finish the PRs in #731 so that you can get the rates you want via `pdf.expected_data(pdf.config.suggested_init(),return_by_sample=True)`

**Comment by alexander-held** [comment]
Right, `pdf.expected_data` is what I'll be using for now. Together with https://github.com/scikit-hep/pyhf/issues/1029#issuecomment-675371293 I can get the behavior I'm looking for.
